appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -456,6 +490,17 @@
                         (cdr event-info)))
                      events))
 
+         ;; Register the multi events with the Simple event dispatcher
+         (define (register-multi-events events)
+           (define-alias SimpleEventDispatcher
+             <com.google.appinventor.components.runtime.EventDispatcher>)
+           (for-each (lambda (eventName)
+                       ;; Tell the Simple event dispatcher to delegate dispatching of this event to this class
+                       (SimpleEventDispatcher:registerMultiEventForDelegation
+                        (as com.google.appinventor.components.runtime.HandlesEventDispatching (this))
+                        eventName))
```

Comment:
Actually, I'm pretty sure that is not a problem. This function is called with a different list of events - the multi events:

(register-multi-events multi-events-to-register)

and this is how that event list is created: 
+       (define (add-to-multi-events event-name)
+         (set! multi-events-to-register
+               (cons event-name multi-events-to-register)))

It is in fact just a list of event names.